app/views/track_mailer/event_digest.text.erb

```diff
@@ -75,4 +75,10 @@
 <% _('Please click on the link below to cancel or alter these emails.') %>
 <%= @unsubscribe_url %>
 
+<%= _('Disable Email Alerts') %>
```

Comment:
Maybe we should call this "Unsubscribe From Email Alerts"? I'm not entirely sure on this but it is the language people seem to use/look for when they're trying to make the emails go away.

Or maybe make a ticket to review the wording in general 🤔

Reply:
I did think about this, but I initially thought that "Disable Email Alerts" makes a better distinction between the two actions. On reflection I think you're right – Unsubscribe is what you look for, and is clear enough. Fixed in cdf9513d844afed5b5d97c3bba4c3f1935441293.

Reply:
I also wasn't sure about this. The intent of disable is clearer but if I'm in a rush and become sufficiently irritated by something I don't remember signing up for to be considering the "Mark as Spam" button, I'll probably be skim-reading looking for "Unsubscribe"